c2corg_ui/templates/xreport/view.html

```diff
@@ -0,0 +1,194 @@
+
+<%block name="metarobots">
+  % if version:
+    <meta name="robots" content="noindex,follow">
+  % else:
+    <meta name="robots" content="index,follow">
+  % endif
+</%block>
+
+<%block name="moduleConstantsValues">
+  <%
+      if xreport['geometry']:
+          geometry = xreport['geometry']['geom_detail'] \
+              if xreport['geometry']['geom_detail'] else xreport['geometry']['geom']
+      else:
+          geometry = None
+  %>
+
+  module.value('mapFeatureCollection',
+  % if geometry:
+    {
+      "type": "FeatureCollection",
+      "properties": {},
+      "features": [
+      {
+        "type": "Feature",
+        "geometry": ${geometry | n},
+        "properties": {
+          "title": ${dumps(locale['title']) | n},
+          "lang": "${locale['lang']}",
+          "documentId": ${xreport['document_id']},
+          % if xreport['elevation'] is not None:
+            "elevation": ${xreport['elevation']},
+          % endif
+          "module": "xreports",
+          "highlight": false
+        }
+      }
+    ]}
+  % else:
+    null
+  % endif
+  );
+
+  module.value('documentData', {
+    "document_id": ${xreport.get('document_id')},
+    "lang": "${lang}",
+    "type": "x",
+    "topic_id": ${dumps(locale.get('topic_id'))}
+    % if not version:
+       , "associations": ${dumps(xreport.get('associations')) | n}
+    % endif
+  });
+</%block>
+
+<header class="view-details-title">
+  <h1 class="routes">
+    <div class="icons-header routes">${get_activities(xreport, 'top')}</div>
+    <span class="title">${show_locale_title(locale)}</span>
+    ${show_badge(xreport, 'xreport')}
+  </h1>
+  <div class="outing-date" ng-cloak>${show_fulldate(xreport['date'])}</div>
+</header>
+
+% if xreport.get('areas'):
+  <article class="location-static text-center">
+    ${show_areas(xreport)}
+    % if xreport.get('elevation'):
+      <p class="text-center"><b>${xreport['elevation']} m</b></p>
+    % endif
+##     TODO SHOW COORDINATES ON THE YELLOW PAPER IN THE HEADER MAP
+##     % if geometry4326:
+##       <p>${round(geometry4326.x)} °E / ${round(geometry4326.y)} °N</p>
+##     % endif
+  </article>
+% endif
+
+<app-map class="view-details-map"></app-map>
+
+<section class="view-details-section images" app-view-details>
+  % if version:
+    ${show_archive_data('xreports', xreport, locale, version)}
+  % endif
+
+  ## IMAGES
+  % if not version:
+    ${get_image_gallery()}
+  % endif
+
+  <div class="view-details-informations col-xs-12 informations">
+    <h3 class="heading informations" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#document-informations">
+      <span translate>Information</span><span class="glyphicon glyphicon-menu-down"></span>
+    </h3>
+    <section id="document-informations" class="collapse in">
+      ${get_xreport_general(xreport)}
+      % if not version:
+        ${get_xreport_participants(xreport)}
+        ${get_xreport_personal(xreport)}
```

Comment:
I don't think you can directly plug the personal info in the mako template. Those data are not public and should only be made available to the xreport author and moderators. But the UI server knows nothing about the current user rights.
It means those data should be added using an additional asynchronous API request depending on the user's permissions (a bit like for profiles).
For the golive, it's OK to simply not show those data at all (comment the line out).